### PR TITLE
Add basic HTML skeleton.

### PR DESCRIPTION
**Pull request by GALemmon**
So it begins!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <p>
+      "test"
+    </p>
+    <script type="text/javascript" src="main.js"></script>
```
Comment (KATIETOLER) [inline index.html:11]: Looks good. Great start, team!

**Review by KATIETOLER — COMMENTED** [review]
Looks good great start team!

**Review by nvalentini21 — COMMENTED** [review]
Looks great! Glad we could do this driver-navigator style.

**Comment by GALemmon** [comment]
Hooray!

**Pull request merged**